### PR TITLE
Add explanations to the thread state tooltips

### PR DESCRIPTION
**Pull request by dpallotti**
They match the explanations in the documentation page for the states that are
mentioned there.

### CODE REVIEW

File: OrbitGl/ThreadStateTrack.cpp [hunk 1]
```diff
@@ -97,6 +97,33 @@ static std::string GetThreadStateName(ThreadStateSliceInfo::ThreadState state) {
...
+    case ThreadStateSliceInfo::kInterruptibleSleep:
+      return "The thread is waiting for a resource to become available or for an event to happen.";
+    case ThreadStateSliceInfo::kUninterruptibleSleep:
+      return "The thread performed a specific system call that cannot be interrupted by any signal "
+             "and is waiting for the call to complete.";
+    case ThreadStateSliceInfo::kStopped:
+      return "The execution of the thread was suspended with the SIGSTOP signal.";
+    case ThreadStateSliceInfo::kTraced:
+      return "The thread is stopped because a tracer (for example, a debugger) is attached to it.";
+    case ThreadStateSliceInfo::kDead:
```
Comment (reichlfl) [inline OrbitGl/ThreadStateTrack.cpp:115]: Indicate the intentional fallthrough here. The style guide suggests `ABSL_FALLTHROUGH_INTENDED;` (https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements).
I guess `[[fallthrough]];` should be supported as well by our compilers.

**Review by reichlfl — APPROVED** [review]
Please add the fallthrough indication as mentioned in the style guide. Apart from that, LGTM!

File: OrbitGl/ThreadStateTrack.cpp (hunk 1 above)
Comment (dpallotti) [reply OrbitGl/ThreadStateTrack.cpp:117]: Good point, there were also a couple missing in a function above, done.

**Review by ronaldfw — APPROVED** [review]
LGTM!

**Pull request merged**